YoutubePlaylistDownloader/GlobalConsts.cs

```diff
@@ -443,7 +452,7 @@ public static async Task TagFile(PlaylistVideo video, int vIndex, string file, F
             if (video == null)
                 throw new ArgumentNullException(nameof(video));
 
-            if (!video.Title.Contains('-'))
+            if (!video.Title.Contains(" - "))
```

Comment:
By the way, the reason for this check is because without it we essentially double the requests to YouTube which could substantially slow down the program. If we check for a title without " - " we can cut that by a lot since we know youtube music videos dont have " - " in the title